### ISSUE TITLE
ERROR: supplicant: wpas (pid:185044) failed unexpectedly, relaunching after short grace period..

### ISSUE THREAD

**Issue by Mangomm** [albfan/miraclecast#493]
```
ERROR: supplicant: wpas (pid:185044) failed unexpectedly, relaunching after short grace period.. (supplicant_failed() in /home/admin1/miracast/miracast-demo/wifi/wifid-supplicant.c:2350)
ERROR: supplicant: wpas (pid:185045) failed again.. entering grace period, waiting 30s before relaunching (supplicant_failed() in /home/admin1/miracast/miracast-demo/wifi/wifid-supplicant.c:2354)
ERROR: supplicant: wpas (pid:185094) failed unexpectedly, relaunching after short grace period.. (supplicant_failed() in /home/admin1/miracast/miracast-demo/wifi/wifid-supplicant.c:2350)
ERROR: supplicant: wpas (pid:185095) failed unexpectedly, relaunching after short grace period.. (supplicant_failed() in /home/admin1/miracast/miracast-demo/wifi/wifid-supplicant.c:2350)
...
```

```
os:
admin1@admin1:~/miracast/miraclecast/src/wifi$ cat /proc/version
Linux version 5.4.18-53-generic (buildd@localhost) (gcc version 9.3.0 (Ubuntu 9.3.0-10kylin2)) #42-KYLINOS SMP Fri Mar 4 03:34:54 UTC 2022
```

when i run:

    systemctl stop NetworkManager.service
    systemctl stop wpa_supplicant.service 

and then，run：

    sudo miracle-wifid & 

It reports an error on the subject above. What is the reason for this?

**Comment by albfan** [comment]
Avoid attach image screenshits if posible.

Ser the wiki FAQ yo attach logs for support

The relevant info will be probably on journalctl

**Comment by Mangomm** [comment]
> 

I have solved the problem, look at the journalctl log, in the /run/miracle/wifi/ path residual stuff, just delete it.
Then the xxx wpa_supplicant or driver does not support P2P xxx problem occurs because the wpa_supplicant service restarts automatically.
Resolved:
1. Stop Internet service first.
systemctl stop NetworkManager.service
systemctl stop wpa_supplicant.service

2. Ensure that all wpa_supplicant services are stopped.
ps -ef | grep wpa_supplican[t]
sudo kill -9 xxx
